testing/test_data.py

```diff
@@ -0,0 +1,29 @@
+from dateutil import parser as dateparser
```

Comment:
Do we need this file anymore with the `factories` in place?

Reply:
No we don't. Removed. 👍